config/application.rb

```diff
@@ -8,6 +8,10 @@
 
 module BooksApp
   class Application < Rails::Application
+    config.i18n.load_path += Dir[Rails.root.join('config', 'locales', '**', '*.{rb,yml}')]
+    config.i18n.default_locale = :ja
+    I18n.available_locales = [:ja, :en]
```

Comment:
```suggestion
    config.i18n.available_locales = [:ja, :en]
```

みたいな書き方はできなかったですか？（できそうな気がするけど未確認）